content/docs/app-developer-guide/specific-buildpacks.md

```diff
@@ -0,0 +1,40 @@
++++
+title="Specify buildpacks"
+weight=1
+summary="Learn how to specify exactly what buildpacks are used during the build process."
++++
+
+{{< param "summary" >}}
+
+### `--buildpacks`
+
+The `--buildpack` parameter can be one of the following:
+
+- path to a directory*, `tar` file, or `tgz` file
+- URL to a `tar` or `tgz` file
+- buildpack located in a builder, in the form of `<id>@<version>`
```

Comment:
`@<version>` is optional no?

Reply:
Yes, you are right. I'll add a note about it.